google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/QueryParameterValue.java

```diff
@@ -384,6 +395,10 @@ public static QueryParameterValue struct(Map<String, QueryParameterValue> struct
         break;
       case STRING:
         return value.toString();
+      case JSON:
+        if (value instanceof String) {
```

Comment:
I wonder if we could also support `Map<String, object>` too? That's likely what we'll be supporting in Python when we implement this.

Reply:
Although  Map<String, Object> is something that would be naturally used in Python, IMO,  JSONObject is more idiomatic in Java. So I'd vote to just keep JSONObject and String.

Reply:
Noted, thanks.